Jenkinsfile

```diff
@@ -15,12 +15,13 @@ pipeline {
         }
     }
     environment {
+        ION_GPG_KEYRING = credentials('ion-releases-key')
         DISABLE_DOWNLOAD_PROGRESS_OPTS = '-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
         LINUX_MVN_RANDOM = '-Djava.security.egd=file:/dev/./urandom'
         COVERAGE_EXCLUSIONS = '**/test/**/*,**/itests/**/*,**/*Test*,**/*.txt,**/*.xml'
         GITHUB_USERNAME = 'connexta'
         GITHUB_REPONAME = 'transformation-api'
-	GITHUB_KEY = 'transformation-api-github-key'
+	    GITHUB_KEY = 'transformation-api-github-key'
```

Comment:
✏️ Extra indent

Reply:
it was missing an indent actually :smile: